frontend/src/static/css/style.scss

```diff
@@ -115,6 +115,52 @@ header {
       margin-bottom: 1em;
       margin-left: 1em;
     }
+
+    .error-message-box {
+      background-color: #F8D7DA;
+      border-color: #F5C6CB;
+      border-radius: 4px;
+      color: #721C24;
+      margin-bottom: 15px;
+      padding: 15px;
+      text-align: left;
+
+      &__close-button {
+        cursor: pointer;
+        float: right;
+        font-size: 25px;
+        font-weight: bold;
+        line-height: 10px;
+
+        &:hover {
+          color: white;
+        }
+      }
+
+      &__message {
+        border-bottom: 2px solid;
+        font-size: .75rem;
```

Comment:
I meant why don't you put the font-size under the parent, so that all child component will inherit by default

Reply:
Oh I thought you meant to eliminate the unnecessary nesting of `&--name`. Will make the changes